apis/profiles.py

```diff
@@ -13,11 +13,20 @@ def get_user_info(username):
     Get the basic info for a user
     TODO cache this?
     '''
-    if (session.get("username") == username or session.get("admin")):
-        query = "SELECT `username`, `email`, `email_confirmed`, `admin` FROM `users` WHERE `username`=%s"
-    else:
-        query = "SELECT `username`, `email_confirmed`, `admin` FROM `users` WHERE `username`=%s"
 
+    admin_query_string = 'admin,' if session.get("username") == username or session.get("admin") else ''
+    query = f'''
+    SELECT
+        username,
+        email,
```

Comment:
We shouldn't expose people's emails

Reply:
that was the point of only querying for email if the user it itself.

Reply:
Oh oops, yeah this was unintentional good catch

Reply:
fixed